ui/main/src/app/services/authentication/authentication.service.ts

```diff
@@ -96,31 +106,41 @@ export class AuthenticationService {
      */
     instantiateAuthModeHandler(mode: string): AuthenticationModeHandler {
         if (mode.toLowerCase() === 'implicit') {
-            this.instantiateImplicitFlowConfiguration();
-            return new ImplicitAuthenticationHandler(this, this.store, sessionStorage);
+            this.implicitConf = {...this.implicitConf, issuer: this.delegateUrl, clientId: this.clientId,clearHashAfterLogin: false};
+            return new ImplicitAuthenticationHandler(this, this.store, sessionStorage,this.oauthService,this.guidService,this.router,this.implicitConf,);
         }
         return new PasswordOrCodeAuthenticationHandler(this, this.store);
     }
 
+
+    regularCheckTokenValidity() {
+        if (this.verifyExpirationDate()) {
+            setTimeout(() => {
+                this.regularCheckTokenValidity();
+            }, 5000);
```

Comment:
This means expiration date is checked every 5s ?
If so a named constant might be better.

Reply:
Yes , i will add a constant